tests/llm/test_anyscale.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from unittest import mock
+
+import pytest
+from distilabel.llm.anyscale import AnyscaleLLM
+from distilabel.tasks.text_generation.base import TextGenerationTask
+
+
+@mock.patch("distilabel.llm.anyscale.OpenAI")
+@mock.patch(
+    "distilabel.llm.anyscale.OpenAILLM.available_models",
+    ["HuggingFaceH4/zephyr-7b-beta"],
+)
+class TestAnyscaleLLM:
+    def test_llm_anyscale(self, mock_anyscale):
+        llm = AnyscaleLLM(
+            model="HuggingFaceH4/zephyr-7b-beta",
```

Comment:
Is there any chance we use `argilla/notus-7b-v1` instead?

Reply:
In the tests there should be no problem, will update it this afternoon. But it's not available to run (I requested both notus and notux, but haven't received an answer yet). This are the supported ones https://docs.endpoints.anyscale.com/category/supported-models.

Reply:
> In the tests there should be no problem, will update it this afternoon. But it's not available to run (I requested both notus and notux, but haven't received an answer yet). These are the supported ones https://docs.endpoints.anyscale.com/category/supported-models.

Updated